test/unit/mdc-slider/mdc-slider.test.js

```diff
@@ -232,6 +236,13 @@ test('#stepDown decrements the slider by the step value if no value given and a
   assert.equal(component.value, 10);
 });
 
+test('adapter#hasClass checks if a class exists on root element', () => {
+  const {component} = setupTest();
+  const hasFoo = component.getDefaultFoundation().adapter_.hasClass('foo');
```

Comment:
See comment above. Remove `foo` from the fixture, and refactor the test to looks something like:

```js
const {root, component} = setupTest();

root.classList.add('foo');

assert.isTrue(component.getDefaultFoundation().adapter_.hasClass('foo');
```